README.md

```diff
@@ -11,7 +11,7 @@ Easily execute the behaviour against matching F# tick methods, attributed C# or
 # Installation
 
 Simply reference TickSpec via [Nuget](https://www.nuget.org/packages/TickSpec/), download the assembly or build the project from source.
-TickSpec works in Visual Studio 2008, 2010 & 2012 and supports both .Net and Silverlight projects.
+TickSpec works in Visual Studio 2017.
```

Comment:
Can you make it say 2015 too  please ?- I will verify. I think that's still an important target fora variety of users.